### ISSUE TITLE
Show Bluetooth permission when tap on SCAN button

### ISSUE THREAD

**Issue by markoaamunkajo** [ruuvi/com.ruuvi.station.ios#1136]
Description:

Bluetooth permission is shown currently during onboarding and especially if user waits on onboarding screens it may even be shown on the first page.

![IMG_0097](https://user-images.githubusercontent.com/50437378/200991064-1cbdd1a0-4312-4d29-987b-940153800250.PNG)

Expected:
Request permission when pressing SCAN button on last onboarding, not before that.

**Comment by markoaamunkajo** [comment]
Closing as duplicate of https://github.com/ruuvi/com.ruuvi.station.ios/issues/1355
